support/mem/configmgr/service-connection-tool-not-download-updates.md

```diff
@@ -31,6 +31,12 @@ This issue occurs in one of the following situations:
 - The specific URLs that are required by the service connection point aren't included in the allowlist on your firewall or proxy server.
 
 ## Resolution
+- If you don't have already, Please consider upgrading the server to Server 2016 or above, 
+- Please make sure you have all the internet rules and exceptions in place for your proxy/firewall to allow the traffic, 
+- Related docs
+[Internet Access Requirement](https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/network/internet-endpoints). 
+[ConfigMgr Proxy Exceptions](https://docs.microsoft.com/en-us/archive/blogs/configmgrdogs/configuration-manager-proxy-exceptions)
+
```

Comment:
Need to add this too

Please make sure below 4 ciphers are enabled on the server with Service Connection Point role
- TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (0xc030)
- TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 (0xc02f)
- TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 (0x9f)  
- TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 (0x9e)  
[Check Ciphers](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/manage-ssl-protocols-in-ad-fs#enabling-or-disabling-additional-cipher-suites)